io.catenax.shared.business_partner_number/RELEASE_NOTES.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+All notable changes to this model will be documented in this file.
+
+## [1.0.0] - 2023-09-11
+### Added
+- initial model
```

Comment:
Can you please also add some more description here about what this model is about.